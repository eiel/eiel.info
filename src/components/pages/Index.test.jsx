/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import Component from './Index';

it('render /', () => {
  const tree = renderer.create(<Component isDev={false} />).toJSON();
  expect(tree).toMatchSnapshot();
});
