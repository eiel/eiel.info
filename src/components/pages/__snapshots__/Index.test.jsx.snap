// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render / 1`] = `
<html
  lang="ja"
>
  <head>
    <meta
      charSet="utf-8"
    />
    <script
      dangerouslySetInnerHTML={
        Object {
          "__html": "(function(w,d,s,l,i){ w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0], j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f); })(window,document,'script','dataLayer', \\"GTM-T7VV63D\\");",
        }
      }
    />
    <meta
      content="ie=edge"
      httpEquiv="x-ua-compatible"
    />
    <title>
      eiel.info
    </title>
    <meta
      content="Tomohiko Himura's Website. He a.k.a. eiel"
      name="description"
    />
    <meta
      content="width=device-width, initial-scale=1"
      name="viewport"
    />
    <link
      href="apple-touch-icon.png"
      rel="apple-touch-icon"
    />
    <link
      href="/main.css"
      rel="stylesheet"
      type="text/css"
    />
  </head>
  <body>
    <noscript>
      <iframe
        height="0"
        src="https://www.googletagmanager.com/ns.html?id=GTM-T7VV63D"
        style={
          Object {
            "display": "none",
            "visibility": "hidden",
          }
        }
        title="GTM"
        width="0"
      />
    </noscript>
    <div
      id="root"
    >
      <div>
        <h1>
          eiel.info
        </h1>
        <ul>
          <li>
            <a
              href="https://github.com/eiel"
            >
              GitHub
            </a>
          </li>
          <li>
            <a
              href="https://twitter.com/eielh"
            >
              Twitter
            </a>
          </li>
          <li>
            <a
              href="https://www.facebook.com/eielh"
            >
              Facebook
            </a>
          </li>
          <li>
            <a
              href="http://qiita.com/eielh"
            >
              Qiita
            </a>
          </li>
          <li>
            <a
              href="https://plus.google.com/+himuratomohiko"
            >
              Google+
            </a>
          </li>
          <li>
            <a
              href="http://eiel.hatenablog.com/"
            >
              Hatena Blog - 補助人工心臓と子育てとリモートワーク
            </a>
          </li>
          <li>
            <a
              href="https://www.flickr.com/photos/163677625@N05/"
            >
              Flickr
            </a>
          </li>
          <li>
            <a
              href="https://www.flickr.com/photos/43922152@N04/"
            >
              旧Flickr
            </a>
          </li>
        </ul>
      </div>
    </div>
  </body>
</html>
`;
